clients/geth/specular/Makefile

```diff
@@ -4,10 +4,18 @@ GOBIN = ./build/bin
 
 install: bindings geth
 
-bindings:
+compile-contracts:
```

Comment:
minor nit: just `contracts` instead of `compile-contracts` (compile implied by make action)